src/main/java/com/manywho/services/azure/facades/AzureFilterBuilder.java

```diff
@@ -0,0 +1,15 @@
+
+package com.manywho.services.azure.facades;
+
+public class AzureFilterBuilder {
+
+	public static String buildFilterExpression(String searchTerm) {
+        String filter = "";
+
+        if(filter == "" && searchTerm != null){
```

Comment:
While `filter == ""` is superfluous here (it will always be `true`), I think what you actually wanted to do is `filter.equals("")` - using `==` to compare objects of type `String` isn't a great idea as it checks to see if the references are equal, rather than the actual content of the string (`String` is an object, not a primitive).

This StackOverflow question gives good information on the topic: https://stackoverflow.com/questions/767372/string-equals-versus